Sources/MongoSwift/MongoCollection+FindAndModify.swift

```diff
@@ -198,6 +198,9 @@ public struct FindOneAndReplaceOptions: FindAndModifyOptionsConvertible, Decodab
     /// Specifies a collation to use.
     public var collation: BSONDocument?
 
+    /// A document or string that specifies the index to use to support the query.
+    public var hint: BSONDocument?
```

Comment:
this option is only supported in MongoDB 4.4+, can we mention so in the docstring? ditto for the option in the other docstring